new/LIT5944Mystagogia.xml

```diff
@@ -0,0 +1,67 @@
+<?xml-model href="https://raw.githubusercontent.com/BetaMasaheft/schema/master/tei-betamesaheft.rng" 
+schematypens="http://relaxng.org/ns/structure/1.0"?><?xml-model href="https://raw.githubusercontent.com/BetaMasaheft/schema/master/tei-betamesaheft.rng" 
+type="application/xml" schematypens="http://purl.oclc.org/dsdl/schematron"?>
+<TEI xmlns="http://www.tei-c.org/ns/1.0" xml:id="LIT5944Mystagogia" xml:lang="en" type="work">
```

Comment:
better xml:id="LIT5944CommMystatgogia"

Reply:
Unfortunately, I already gave the name LIT5944Mystagogia to the new file.

Reply:
I repeat. As long as a PR has not been merged - since this file was not created through the app - you can move, rename, delete, do whatever you want.

Reply:
OK. I changed the file to the old name and saved it to the folder '5001-6000'. I deleted the branch EMIP00634.